docs/source/reference/storage.rst

```diff
@@ -244,7 +244,21 @@ To manage persistent Storage objects, the sky CLI provides two useful commands -
     NAME               CREATED     STORE  COMMAND                                        STATUS
     sky-dataset-romil  3 mins ago  S3     sky launch -c demo examples/storage_demo.yaml  READY
 
-2.  :code:`sky storage delete` allows you to delete any Storage objects managed
+2.  :code:`sky storage create` allows you to create a sky Storage object. Common use cases include:
+   - Create a storage object by uploading contents from local source path to specified stores
+        :code:`$ sky storage create imagenet -src /Users/skypilotUser/Project -s S3 -s GCS`
+   - Create an empty storage object with GCS stores
+        :code:`$ sky storage create emptyStore -s GCS`
+   - Create a storage object by uploading local source contents to default S3 store
+        :code:`$ sky storage create cleanedData /Users/skypilotUser/Project/CleanedData`
+   - Create a storage object from bucket URL using source bucket to infer name and as the backing store
+        :code:`$ sky storage -src S3://bucket-name`
```

Comment:
Add code comment saying the bucket will be named `bucket-name`

Reply:
Can't be public bucket

Reply:
Mention in Line 254: `Create's a storage object of name `bucket-name`, which is inferred from the bucket's URL. The bucket must be private bucket.

Reply:
Mention that external public buckets are okay and that a user's publically accessible bucket is okay as well.